src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -670,7 +670,7 @@ void DatabaseCatalog::updateDatabaseName(const String & old_name, const String &
             assert(tables_from.size() == 1);
             const auto & the_table_from = *tables_from.begin();
 
-            view_dependencies.removeDependency(the_table_from, StorageID{old_name, table_name});
```

Comment:
As far as I see, we always call `removeDependency` with `remove_isolated_tables = true`. What do you think about either removing `remove_isolated_tables` or changing its default value to make `TablesDependencyGraph` less bugphrone?